tests/js/spec/components/avatar.spec.jsx

```diff
@@ -136,5 +136,28 @@ describe('Avatar', function() {
       expect(avatar.find('LetterAvatar').prop('displayName')).toBe('test organization');
       expect(avatar.find('LetterAvatar').prop('identifier')).toBe('test-organization');
     });
+
+    it('displays platform list icons for project Avatar', function() {
+      let project = TestStubs.Project({
+        platforms: ['python', 'javascript'],
+        platform: 'java',
+      });
+      let avatar = mount(<Avatar project={project} />);
+      expect(avatar.find('PlatformList').prop('platforms')).toEqual([
+        'python',
+        'javascript',
+      ]);
+    });
+
+    it('displays a fallback platform list for project Avatar using the `platform` specified during onboarding', function() {
```

Comment:
did you mean to put backticks here?

Reply:
yeah, since it's semi referring to the property name